cors/resources/expose-headers.py

```diff
@@ -1,10 +1,10 @@
 def main(request, response):
     response.add_required_headers = False
-    output =  "HTTP/1.1 221 ALL YOUR BASE BELONG TO H1\r\n"
-    output += "Access-Control-Allow-Origin: *\r\n"
-    output += "BB-8: hey\r\n"
-    output += "Content-Language: mkay\r\n"
-    output += request.GET.first("expose") + "\r\n"
-    output += "\r\n"
+    output =  u"HTTP/1.1 221 ALL YOUR BASE BELONG TO H1\r\n"
+    output += u"Access-Control-Allow-Origin: *\r\n"
+    output += u"BB-8: hey\r\n"
+    output += u"Content-Language: mkay\r\n"
+    output += request.GET.first(b"expose").decode("iso-8859-1") + u"\r\n"
+    output += u"\r\n"
```

Comment:
This output seems better to use raw bytes, as it's literally the raw bytes on wire. And you won't need to call `decode`.